src/sys.zig

```diff
@@ -2233,27 +2233,14 @@ pub fn isExecutableFileOSPath(path: bun.OSPathSliceZ) bool {
         // Rationale: `GetBinaryTypeW` does not work on .cmd files.
         // Windows does not have executable permission like posix does, instead we
         // can just look at the file extension to determine executable status.
-        @compileError("Do not use isExecutableFilePath on Windows");
-
-        // var out: windows.DWORD = 0;
-        // const rc = kernel32.GetBinaryTypeW(path, &out);
-
-        // const result = if (rc == windows.FALSE)
-        //     false
-        // else switch (out) {
-        //     kernel32.SCS_32BIT_BINARY,
-        //     kernel32.SCS_64BIT_BINARY,
-        //     kernel32.SCS_DOS_BINARY,
-        //     kernel32.SCS_OS216_BINARY,
-        //     kernel32.SCS_PIF_BINARY,
-        //     kernel32.SCS_POSIX_BINARY,
-        //     => true,
-        //     else => false,
-        // };
-
-        // log("GetBinaryTypeW({}) = {d}. isExecutable={}", .{ bun.fmt.utf16(path), out, result });
-
-        // return result;
+        const stringPath = bun.String.createFromOSPath(path);
```

Comment:
It's my first time writing zig. This works but I'm sure there's a better way to do this. Windows 16bit-ish paths that couldn't be squished to utf8 would not match any executable extensions.
![image](https://github.com/oven-sh/bun/assets/13159458/23d6a0d2-0efe-4e6a-8cfd-9b91c4b72f76)